RELEASING/README.md

```diff
@@ -48,53 +48,96 @@ need to be done at every release.
     svn commit -m "Add PGP keys of new Superset committer"
 ```
 
-## Crafting tarball and signatures
+## Setting up the release environment (do every time)
+
+As the vote process takes a minimum of 72h (community vote) + 72h (IPMC) vote,
+often stretching over several weeks calendar time if votes don't pass, chances are
+the same terminal session won't be used for crafting the release candidate and the
+final release. Therefore, it's a good idea to do the following every time you
+work on a new phase of the release process to make sure you aren't releasing
+the wrong files/using wrong names:
 
-Now let's craft a source release
 ```bash
-    # Assuming these commands are executed from the root of the repo
-    export REPO_DIR=$(pwd)
-    # Set VERSION to the release being prepared (rc1 for first vote on version)
-    export VERSION=0.34.1rc1
+    # Set VERSION to the release being prepared, e.g. 0.34.1.
+    export VERSION=XX.YY.ZZ
```

Comment:
not in scope for this review, but I feel like we should prefix these env vars with `SUPERSET` (e.g. `SUPERSET_VERSION` and `SUPERSET_RC`) since they could easily conflict with something else in the user's terminal session.

Reply:
Agree here too.